runtime/vm/JVMImage.cpp

```diff
@@ -384,6 +384,96 @@ JVMImage::fixupClasses(void)
 	}
 }
 
+void
+JVMImage::fixupClass(J9Class *clazz)
+{
+	clazz->classObject = NULL;
+	fixupMethodRunAddresses(clazz);
+	fixupConstantPool(clazz);
+	clazz->initializeStatus = J9ClassInitNotInitialized;
+	clazz->jniIDs = NULL;
+	clazz->replacedClass = NULL;
+	clazz->callSites = NULL;
+	clazz->methodTypes = NULL;
+	clazz->varHandleMethodTypes = NULL;
+	clazz->gcLink = NULL;
+
+	UDATA totalStaticSlots = totalStaticSlotsForClass(clazz->romClass);
+	memset(clazz->ramStatics, 0, totalStaticSlots * sizeof(UDATA));
+
+	UDATA i;
+	if (NULL != clazz->staticSplitMethodTable) {
+		for (i = 0; i < clazz->romClass->staticSplitMethodRefCount; ++i) {
```

Comment:
Not specific to your code. But not all of our for loops pre-increment
Some of our loops use post-increment

Not a huge deal but we should try and be consistent if we can.
Considering they are both the same (at least in a for-loop context). You can choose the style :)